src/components/Modals/SetAclPermsDocContainerModal.jsx

```diff
@@ -0,0 +1,172 @@
+// React Imports
+import React, { useContext, useState } from 'react';
+// Custom Hook Imports
+import { useSession, useStatusNotification } from '@hooks';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import Dialog from '@mui/material/Dialog';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import SettingsIcon from '@mui/icons-material/Settings';
+import TextField from '@mui/material/TextField';
+// Utility Imports
+import { runNotification, setDocContainerAclPermission } from '@utils';
+// Context Imports
+import { SignedInUserContext } from '@contexts';
+// Component Imports
+import FormSection from '../Form/FormSection';
+
+/**
+ * @typedef {import("../../typedefs.js").setAclPermsDocContainerModalProps} setAclPermsDocContainerModalProps
```

Comment:
Similar to the comment on the `DocumentTable`, we would need to create a new typedef for `setAclPermsDocContainerModalProps` for the component in `typedefs.js` for these imports. (see comment before for guidance)